instrumentation/log4j/log4j-appender-2.17/library/README.md

```diff
@@ -56,3 +56,45 @@ The following demonstrates how you might configure the appender in your `log4j.x
 
 In this example Log4j2 log events will be sent to both the console appender and
 the `OpenTelemetryAppender`.
+
+In order to function, `OpenTelemetryAppender` needs access to an `OpenTelemetry` instance. This can
+be programmatically set during application startup as follows:
+
+```java
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.core.LoggerContext;
+import org.apache.logging.log4j.core.config.Configuration;
+import io.opentelemetry.instrumentation.log4j.appender.v2_17.OpenTelemetryAppender;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+
+public class Application {
+
+  public static void main(String[] args) {
+    OpenTelemetrySdk openTelemetrySdk = // Configure OpenTelemetrySdk
+
+    // Get Configuration, iterate through appenders, and call setOpenTelemetrySdk(...) on any OpenTelemetryAppender
+    Configuration config = ((LoggerContext) LogManager.getContext(false)).getConfiguration();
```

Comment:
Would it make sense to have this code in a static method of `OpenTelemetryAppender` instead? Using a static method to set up the appender would be similar to jul to slf4j bridge using `SLF4JBridgeHandler.install()`

Reply:
Yeah I think that could make sense. I wonder if there's any edge cases where a standard `install(..)` method would struggle to find the the OpenTelemetryAppender....

Still probably worth it, even if edge cases exist.

Reply:
Pushed a commit that does this. Really cleans up the user experience and IMO makes this approach the clear winner.